api/swagger.yaml

```diff
@@ -8263,3 +8263,62 @@ paths:
           format: "int64"
           required: true
       tags: ["Secret"]
+  /distribution/{name}/json:
```

Comment:
Yup, and that's one thing the API currently doesn't do 😞 - think there was an attempt once to change it, but it would break existing clients